static/app/components/quickTrace/styles.tsx

```diff
@@ -54,12 +53,6 @@ export const EventNode = styled(Tag)<{shouldOffset?: boolean}>`
     background-color: ${p => nodeColors(p.theme)[p.type || 'white'].background};
     border: 1px solid ${p => nodeColors(p.theme)[p.type || 'white'].border};
   }
-
-  /*
-   * When the EventNode is contains an icon, we need to offset the
-   * component a little for all the EventNodes to be aligned.
-   */
-  ${p => p.shouldOffset && `margin-top: ${space(0.5)}`}
```

Comment:
We no longer need a `shouldOffset` prop to realign the contents `EventNode` (created as a styled `Tag`), since `Tag` now uses flex layout to center its inner content.